builtin/files/stack-templates/control-plane.json.tmpl

```diff
@@ -385,6 +385,11 @@
                 },
                 {{end}}
                 {
```

Comment:
This is going to need a `,` after it.

Reply:
I think it was just bad formatting wi the spacing.